plugin/path_rotate_root_creds.go

```diff
@@ -21,6 +21,11 @@ func (b *backend) pathRotateCredentials() *framework.Path {
 				ForwardPerformanceStandby:   true,
 				ForwardPerformanceSecondary: true,
 			},
+			logical.UpdateOperation: &framework.PathOperation{
```

Comment:
I think you'll also need `logical.CreateOperation` to point to the same callback as well. Similar to OpenLDAP: https://github.com/hashicorp/vault-plugin-secrets-openldap/blob/master/path_rotate.go#L26-L35

Reply:
Done

Reply:
Since there is no existence check, the CreateOperation will never be called.

Reply:
Oh I didn't know that, nice! @Valarissa can probably remove it then.